CHANGELOG.md

```diff
@@ -14,10 +14,21 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Removed
 
+### Fixed
+
+### Security
+
+## 2.4.2 - 2025-01-02
```

Comment:
I'd like to make a new civis-python release (for a sec update, also the updated Civis API spec), but I didn't want to do so between Xmas and New Years. I also don't think we'd need new releases for datascience-python and civis-jupyter-python3.